go/nbs/store.go

```diff
@@ -313,6 +315,13 @@ func (nbs *NomsBlockStore) AddHints(hints types.Hints) {
 }
 
 func (nbs *NomsBlockStore) Flush() {
-	success := nbs.UpdateRoot(nbs.root, nbs.root)
-	d.Chk.True(success)
+	b := &backoff.Backoff{
+		Min:    50 * time.Millisecond,
+		Max:    10 * time.Second,
```

Comment:
I have no real basis for this maximum. I set the minimum at 50ms because I vaguely remembered that being the same order of magnitude as a dynamoDB write. Please feel free to suggest other numbers.